build.gradle

```diff
@@ -51,51 +50,21 @@ configure(allprojects) {
 	apply plugin: "io.spring.dependency-management"
 
 	ext {
-		springBootVersion = project.findProperty("springBootVersion") ?: "2.1.4.RELEASE"
-		springVersion = project.findProperty("springVersion") ?: "5.1.6.RELEASE"
-		reactorVersion = project.findProperty("reactorVersion") ?: "Californium-SR6"
-		openServiceBrokerVersion = "3.0.0.RELEASE"
+		springBootVersion = "2.1.8.RELEASE"
```

Comment:
We should keep these overrides. It's not overriding anything by default, but it's allowing us to override the version in CI or for testing. For example, we have a nightly CI build for spring cloud open service broker that builds and verifies compatibility with the latest boot and framework snapshots.

Reply:
That's true but we are not using them at the moment. I am happy to create a chore to create the concourse CI job for it.
What do you think @royclarkson?

Reply:
Good idea. We'll need to keep the 1.0.x releases of App Broker on Boot 2.1.x, so maintaining a nightly CI build that verifies everything works on the latest releases or snapshots is pretty valuable.

Reply:
Here it is: https://github.com/spring-cloud/spring-cloud-app-broker/issues/274